### PR TITLE
Provide the new nginx-controller images

### PR DESCRIPTION
**Pull request by manuelbuil**
Signed-off-by: Manuel Buil <mbuil@suse.com>

### CODE REVIEW

File: stable/nginx-ingress/values.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 controller:
   name: controller
   image:
-    repository: registry.suse.com/sles12/nginx-ingress-controller
-    tag: "0.15.0"
+    repository: registry.suse.de/devel/caasp/4.5/containers/containers/caasp/v4.5/ingress-nginx-controller
```
Comment (thehejik) [inline stable/nginx-ingress/values.yaml:7]:
```suggestion
    repository: registry.suse.com/caasp/v4.5/ingress-nginx-controller
```

File: stable/nginx-ingress/values.yaml [hunk 2]
```diff
@@ -448,8 +448,8 @@ defaultBackend:
 
   name: default-backend
   image:
-    repository: registry.suse.com/sles12/default-http-backend
-    tag: "0.15.0"
+    repository: registry.suse.de/devel/caasp/4.5/containers/containers/caasp/v4.5/default-http-backend
```
Comment (thehejik) [inline stable/nginx-ingress/values.yaml:451]:
```suggestion
    repository: registry.suse.com/caasp/v4.5/default-http-backend
```

File: stable/nginx-ingress/values.yaml (hunk 1 above)
Comment (thehejik) [reply stable/nginx-ingress/values.yaml:7]: We should switch to `registry.suse.com` once the images are released.

**Comment by dirkmueller** [comment]
see #174 for a fixed chart

**Comment by dirkmueller** [comment]
I believe that there's no additional change here anymore over what has been merged via #174 . Closing. Please don't hesitate to reopen if I missed something!

**Pull request closed without merging**